### ISSUE TITLE
Smart UV Sync keybind is disabled in addon options but it still activates on Right Mouse

### ISSUE THREAD

**Issue by SvenViland** [Dangry98/Key-Ops-Toolkit#2]
https://github.com/user-attachments/assets/102b3249-029e-4451-a272-7369242cc9d1

Blender 4.2.0

**Comment by Dangry98** [comment]
Thank you for reporting! I look into it when I get time :)

**Comment by Dangry98** [comment]
Hi, I tried to look into this, but unable to replicate, make sure you have not added it yourself or its another add-on.

**Comment by SvenViland** [comment]
You are correct. For some reason I had a duplicate toggle sync keybind on right mouse which wasn't disabled. Thanks for your help.

**Comment by Dangry98** [comment]
Alright, closing the issue :)
